src/app/evaluate-absences/components/evaluate-absences-list/evaluate-absences-list.component.ts

```diff
@@ -59,16 +58,16 @@ export class EvaluateAbsencesListComponent implements OnInit, AfterViewInit {
       .pipe(take(1))
       .subscribe((filterValue) => this.state.setFilter(filterValue));
 
-    // Remove Column TotalHalfDays if the corresponding PresenceType is inactive
+    // Add Column TotalHalfDays if the corresponding PresenceType is active
     this.presenceTypes$
       .pipe(
         map((types) =>
-          types.filter((t) => isHalfDay(t, this.settings) && t.Active)
+          Boolean(types.find((t) => isHalfDay(t, this.settings))?.Active)
         )
       )
-      .subscribe((types) => {
-        if (types.length === 0) {
-          this.columns = this.columns.filter((c) => c.key !== 'TotalHalfDays');
+      .subscribe((activeHalfDay) => {
+        if (activeHalfDay) {
+          this.columns.push({ key: 'TotalHalfDays', label: 'halfday' });
```

Comment:
Du kannst den `ReadonlyArray` behalten und immutable arbeiten: `this.columns = [...this.columns, { key: 'TotalHalfDays', label: 'halfday' }]`